File: testdata/jennies/rawtypes/struct_with_complex_fields/PHPRawTypes/src/Types/StructComplexFields/SomeOtherStruct.php
<?php

namespace Grafana\Foundation\Types\StructComplexFields;

class SomeOtherStruct implements \JsonSerializable {
    /**
     * @var mixed
     */
    public $fieldAny;

    /**
     * @return array<string, mixed>
     */
    public function jsonSerialize(): array
    {
        $data = [
            "FieldAny" => $this->fieldAny,
        ];
        return $data;
    }
}

File: testdata/jennies/rawtypes/disjunctions/CSharpRawTypes/src/Grafana/Foundation/Disjunctions/SomeOtherStruct.cs
namespace Grafana.Foundation.Disjunctions;


public class SomeOtherStruct
{
    public string Type;
    public byte Foo;

    public SomeOtherStruct()
    {
        this.Type = "";
        this.Foo = (byte) 0;
    }

    public SomeOtherStruct(string type, byte foo)
    {
        this.Type = type;
        this.Foo = foo;
    }
}
